cmd/scheduler.go

```diff
@@ -77,17 +77,102 @@ func getImqsHttpPort() int {
 
 func setDefaultVariables() {
 	imqsHttpPort := getImqsHttpPort()
-	config.Variables = make(map[string]string)
-	config.Variables["LOCATOR_SRC"] = "c:\\imqsvar\\imports"
-	config.Variables["LEGACY_LOCK_DIR"] = "c:\\imqsvar\\locks" // No longer needed, since we serialize all scheduled tasks. Should remove from imqstool.
-	config.Variables["JOB_SERVICE_URL"] = "http://localhost"
+	staticConfig.Variables = make(map[string]string)
+	staticConfig.Variables["LOCATOR_SRC"] = "c:\\imqsvar\\imports"
+	staticConfig.Variables["LEGACY_LOCK_DIR"] = "c:\\imqsvar\\locks" // No longer needed, since we serialize all scheduled tasks. Should remove from imqstool.
+	staticConfig.Variables["JOB_SERVICE_URL"] = "http://localhost"
 	if imqsHttpPort != 80 {
-		config.Variables["JOB_SERVICE_URL"] = config.Variables["JOB_SERVICE_URL"] + ":" + strconv.Itoa(imqsHttpPort)
+		staticConfig.Variables["JOB_SERVICE_URL"] = staticConfig.Variables["JOB_SERVICE_URL"] + ":" + strconv.Itoa(imqsHttpPort)
 	}
 }
 
-func addCommands() {
-	add := func(enabled bool, name, pool string, interval time.Duration, timeout time.Duration, exec string, params ...string) *scheduler.Command {
+func overlayClientConfig() {
```

Comment:
This is messy. There is a lot of duplication of what load from static... does.  It will be much cleaner if you read both sets of config, merge them into a unified list of tasks and then add them